### PR TITLE
Added metadata to monorepo, and also updated it reference into other package.json

### LINKED ISSUE #1366: Add metadata to monorepo and make it an npm package

### PR DESCRIPTION
**Pull request by msoni89**

### CODE REVIEW

File: packages/metadata/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This script builds the files in the main and module directories.
+
+rm main/networks/list.cjs
+rm module/networks/list.js
+
+touch main/networks/list.cjs
+touch module/networks/list.js
+
+echo '/* eslint-disable */
```
Comment (0xdavinchee) [inline packages/metadata/build.sh]: note: this is added here to prevent eslint from complaining

Comment (hellwolf) [reply packages/metadata/build.sh]: use idiomatic way (aka. heredoc) instead:

```
cat > module/networks/list.js <<EOF
... the content ...
EOF
```

**Comment by hellwolf** [comment]
bootstrapped: https://www.npmjs.com/package/@superfluid-finance/metadata

**Comment by hellwolf** [comment]
this https://github.com/superfluid-finance/protocol-monorepo/pull/1498/files will further full automate the release process.

**Review by hellwolf — APPROVED** [review]

**Pull request merged**